### PR TITLE
feat(model) : Add Attention U-net model

### PR DESCRIPTION
**Pull request by ash-01xor**
Implemented the Attention U-net model and made changes such that it adheres to the existing codebase 

Overview of the changes made in the pull request:
- Added new model
- written the attention block which will be used inside the decoder in this model

Thanks for your time and assistance, looking forward to collaborating more.

### CODE REVIEW

**Review by ayyucedemirbas — COMMENTED** [review]
looks great 😊

**Review by ayyucedemirbas — COMMENTED** [review]
I get this error message:

`AttributeError: module 'keras.api._v2.keras' has no attribute 'layer'`

File: kerasfuse/models/attention_unet.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        input2 (Tensor): Second input tensor.
+
+    Returns:
+        output (Tensor): Output tensor after applying attention.
+    """
+    g = tf.keras.layers.Conv2D(filters=input1.shape[-1], kernel_size=1)(input1)
+    x = tf.keras.layers.Conv2D(filters=input2.shape[-1], kernel_size=1)(input2)
+    psi = tf.keras.layers.Add()([g, x])
+    psi = tf.keras.layers.Activation("relu")(psi)
+    psi = tf.keras.layer.Conv2D(filters=1, kernel_size=1)(psi)
```
Comment (ayyucedemirbas) [inline kerasfuse/models/attention_unet.py]: this must be `psi = tf.keras.layers.Conv2D(filters=1, kernel_size=1)(psi)`

**Review by ayyucedemirbas — CHANGES_REQUESTED** [review]
please change the line 19 to `psi = tf.keras.layers.Conv2D(filters=1, kernel_size=1)(psi)`

File: kerasfuse/models/attention_unet.py (hunk 1 above)
Comment (ash-01xor) [reply kerasfuse/models/attention_unet.py]: Have made the necessary changes, must look good now.

**Review by onuralpszr — APPROVED** [review]
LGTM 

Thank you 👍

**Review by onuralpszr — APPROVED** [review]

**Pull request merged**